package.json

```diff
@@ -4,18 +4,16 @@
   "description": "Create and package prebuilds for native modules",
   "main": "index.js",
   "dependencies": {
-    "execspawn": "^1.0.1",
-    "mkdirp-classic": "^0.5.3",
-    "node-abi": "^3.3.0",
-    "npm-run-path": "^3.1.0",
-    "minimist": "^1.2.5",
+    "node-abi": "^3.56.0",
```

Comment:
I suggest separating the bump of node-abi into its own PR, which can be quickly merged as it is currently blocking the support for Node.js v21.

Reply:
I have opened a PR: #87